packages/commerce-sdk-react/src/hooks/useCommerceApi.test.tsx

```diff
@@ -6,27 +6,17 @@
  */
 import 'react'
 import useCommerceApi from './useCommerceApi'
-import {renderHookWithProviders} from '../test-utils'
-import {ApiClients} from './types'
+import {API_CLIENTS_KEYS, renderHookWithProviders} from '../test-utils'
 
 jest.mock('../auth/index.ts')
 
 describe('useCommerceApi', () => {
     test('returns a set of api clients', () => {
-        const clients: (keyof ApiClients)[] = [
-            'shopperBaskets',
-            'shopperContexts',
-            'shopperCustomers',
-            'shopperGiftCertificates',
-            'shopperLogin',
-            'shopperOrders',
-            'shopperProducts',
-            'shopperPromotions',
-            'shopperSearch'
-        ]
         const {result} = renderHookWithProviders(() => useCommerceApi())
-        clients.forEach((name) => {
-            expect(result.current[name]).toBeDefined()
+        const apiClients = result.current
+
+        API_CLIENTS_KEYS.forEach((key) => {
+            expect(apiClients[key]).toBeDefined()
```

Comment:
Making sure that the list of api clients' keys is up to date. It now includes the latest `shopperSeo`.